training/common/usr/libexec/upgrade-informer

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Run the command and capture its output
+output=$(bootc upgrade --check | sed -e 1q)
+message_file="/etc/motd.d/upgrade-message"
+bootc_auth="/etc/ostree/auth.json"
+
+if [[ $output == Update\ available* ]]; then
```

Comment:
we need to get proper support into bootc so we dont have to parse strings.

Reply:
Hmm, didn't find a good way to do it rather this one. I assume i can look on bootc and add , --json maybe?

Reply:
There is no good way to do it, bootc upgrade --check doesn't have an option for json format. 
Though there is draft pr for it  https://github.com/containers/bootc/pull/472#issuecomment-2061337390

Reply:
Why don't we automatically download the image, and not reboot?

Reply:
Hmm, we can run "bootc upgrade" without applying the change but it means that next reboot user will get new version and if he doesn't want? 
We talked about updating user that there is an available upgrade but not applying anything by ourselves for now at least

Reply:
@jeremyeder WDYT?

Reply:
showing folks and update is available is all we set out to do here; over time we can gauge what level of effort to put on this.  we want to nudge users to upgrade for all the normal reasons, but going beyond that feels premature.